Signal/src/view controllers/InboxTableViewCell.m

```diff
@@ -50,9 +50,6 @@ -(void)configureWithThread:(TSThread*)thread {
         if([_nameLabel.text length]==0) {
             _nameLabel.text = @"New Group";
         }
-        if(_contactPictureView.image!=nil) {
```

Comment:
if you test this, this will result in avatars not being circular. 
https://github.com/WhisperSystems/Signal-iOS/blob/797492fc130001e67b36612bf78db434ca462455/Signal/src/util/UIUtil.m
you simply probably need to change the border to be clear. that said, PRs with code changes at this stage aren't super helpful as they just add more work prior to the beta for review